src/main/java/uk/gov/hmcts/darts/task/runner/impl/ProcessDailyListAutomatedTask.java

```diff
@@ -24,13 +29,17 @@ public String getTaskName() {
 
     @Override
     protected void runTask() {
-        // TODO this is to be populated as a part of jira ticket https://tools.hmcts.net/jira/browse/DMP-183
+        dailyListProcessor.processAllDailyLists(LocalDate.now());
     }
 
     @Override
     protected void handleException(Exception exception) {
-        // TODO this is to be populated as a part of jira ticket https://tools.hmcts.net/jira/browse/DMP-183
         log.error("Exception: {}", exception.getMessage());
     }
 
+    @Autowired
+    public void setDailyListProcessor(DailyListProcessor dailyListProcessor) {
```

Comment:
Would this be better if it was passed in via the constructor and then it can be made final?